third_party/blink/renderer/core/dom/attribute.h

```diff
@@ -27,7 +27,9 @@
 #define THIRD_PARTY_BLINK_RENDERER_CORE_DOM_ATTRIBUTE_H_
 
 #include "build/build_config.h"
+#include "base/record_replay.h"
 #include "third_party/blink/renderer/core/dom/qualified_name.h"
+#include "third_party/blink/renderer/core/html_names.h"
 #include "third_party/blink/renderer/platform/wtf/allocator/allocator.h"
```

Comment:
Can we move these new includes to the end of the `#include` section, after all upstream includes?  I used to put the includes we added in sorted order like they are upstream, but it leads to extra merge conflicts when we rebase.

Reply:
@bhackett1024 Oh that's good to know (about the conflict potential). I will try my best to keep that order accordingly. The main problem here is that the default formatter in VSCode will re-order them aggressively regardless (at least in some scenarios, I think after hitting Enter on a nearby line or somesuch, I don't even know when exactly). We probably want to remove that from the standard formatting rules then? Not sure if you have such a problem with your IDE?